docs/sources/release-notes/v2.2.md

```diff
@@ -25,15 +25,27 @@ This feature still needs additional testing; we do not recommend using it in pro
   This feature is experimental and disabled by default; enable it using the flag `-blocks-storage.bucket-store.index-header.map-populate-enabled`.
 
 - **Faster ingester startup**: Ingesters now replay their Write-Ahead-Logs about 50% faster, and they also re-join the ring sooner under some conditions.
 
+- **Helm Chart improvements**: The [Mimir Helm chart](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed) is our recommended way to install Mimir on Kubernetes. As part of the Mimir 2.2 release, we're also releasing a version 3.0 of the Helm chart. Below we mention some of the most notable enhancements. For the full list, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+  - The Helm chart now supports [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift). 
+  - The Helm chart can now easily deploy the [Grafana Agent](https://github.com/grafana/agent) to scrape metrics and logs from all Mimir pods and ship them to a remote store, making it easy to monitor the health of your Mimir installation. For more information on how to set this up, see [Collecting metrics and logs from Grafana Mimir]({{< relref "../operators-guide/monitoring-grafana-mimir/collecting-metrics-and-logs.md" >}}).
+  - The Helm chart now enables multi-tenancy by default. This makes it easy for users to add tenants as they grow their cluster and take advantage of Mimir's per-tenant quality-of-service features, which improve its stability and resilience at high scale. To learn more about how multi-tenancy in Mimir works, see [Grafana Mimir authorization and authentication]({{< relref "../operators-guide/securing/authentication-and-authorization" >}}). We've implemented this change in a way that is backwards compatible. To read about how we did this, see [#2117](https://github.com/grafana/mimir/pull/2117). 
+  - We've significantly leveled up the configuration experience for the Helm chart. A few of the most salient changes:
+    - We've added an `extraEnvFrom` capability to all Mimir services to enable you to inject secrets via environment variables.
+    - We've made it possible to globally set environment variables and inject secrets across all pods in the chart using `global.extraEnv` and `global.extraEnvFrom`. Note that the memcached and minio pods are not included.
+    - We've switched the default storage of the Mimir configuration from a `Secret` to a `ConfigMap`, which makes it easier to quickly diff your Mimir configuration between upgrades. We especially like the [Helm diff](https://github.com/databus23/helm-diff) plugin for this purpose.
+    - We've added a `structuredConfig` option which allows you to overwrite specific key<>value pairs in the `mimir.config` template, saving you from having to maintain the entire `mimir.config` in your own `values.yaml`.
```

Comment:
```suggestion
    - We've added a `structuredConfig` option, which allows you to overwrite specific key<>value pairs in the `mimir.config` template, saving you from having to maintain the entire `mimir.config` in your own `values.yaml`.
```
What is "key<>value". Can we simply write "key-value" in plain English? If not, use `key<>value` instead.